amun/entrypoint.py

```diff
@@ -59,7 +61,8 @@
 application.secret_key = Configuration.APP_SECRET_KEY
 
 # static information as metric
-metrics.info("amun_api_info", "Amun API info", version=__amun_version__)
+metrics.info("amun_api_info", "Amun API info", version=__service_version__)
+_API_GAUGE_METRIC = metrics.info("amun_api_schema_up2date", "Amun API schema up2date")
```

Comment:
This metric does not seem to be needed. Amun does not talk to the knowledge graph so no check for schema is needed.

Reply:
I deleted this line?